### PR TITLE
Update Dart formatter message

### PR DESCRIPTION
**Pull request by mit-mit**
Related to https://github.com/dart-lang/dart_style/issues/986

### CODE REVIEW

**Review by isoos — APPROVED** [review]

**Comment by mit-mit** [comment]
Do we not have a flutter packages golden file? I wanted to make sure the conditional for `dart format`/`flutter format` works as expected.

**Comment by isoos** [comment]
> Do we not have a golden file for flutter package? I wanted to make sure the conditional for dart format/flutter format works as expected.

Yeah, but it is well formatted:
https://github.com/dart-lang/pana/blob/master/test/goldens/end2end/url_launcher-6.0.3.json_report.md

You may test one locally by running `bin/pana.dart --hosted <package> [<version>]`

File: lib/src/messages.dart [hunk 1]
```diff
@@ -11,8 +11,8 @@ class _MsgDict {
   _MsgDict._({
     this.dartanalyzerShortName = 'dart analyze',
     this.dartanalyzerShortCmd = 'dart analyze .',
-    this.dartfmtShortName = 'dartfmt',
-    this.dartfmtShortCmd = 'dartfmt -n .',
+    this.dartfmtShortName = 'dart format',
```
Comment (sigurdm) [inline lib/src/messages.dart:14]: I have a feeling these are unused. @isoos can these be removed?

Comment (mit-mit) [reply lib/src/messages.dart:14]: I think they are used? 

https://github.com/dart-lang/pana/search?q=runningDartanalyzerFailed

Comment (sigurdm) [reply lib/src/messages.dart:14]: https://github.com/dart-lang/pana/search?q=dartfmtShortName shows no usage sites...

Comment (mit-mit) [reply lib/src/messages.dart]: Ah, sorry, I misunderstood. Took at stab at updating the PR. Let's see if that passes CI...

**Review by sigurdm — APPROVED** [review]
LGTM

Thanks for updating the CONTRIBUTION.md!

**Pull request merged**